src/main/kotlin/no/nav/familie/ef/sak/beregning/BeregningController.kt

```diff
@@ -53,4 +54,11 @@ class BeregningController(
             ?: error("Fant ingen startdato for vedtak på behandling med id=$behandlingId")
         return Ressurs.success(tilkjentYtelseService.hentForBehandling(behandlingId).tilBeløpsperiode(startDatoForVedtak))
     }
+
+    @GetMapping("/{behandlingId}/forrigeInntektsgrunnlag")
+    fun hentSisteInntektFraForrigeIverksatteBehandling(@PathVariable behandlingId: UUID): Ressurs<BigDecimal> {
```

Comment:
Trenger vi en metode som henter data til "forrige"? 

Tenker frontend alltid har en referanse til "forrigeBehandlingId" fra behandlingen, sånn at frontend kan kalle backend med "forrigeBehandlingId" og få data direkte til forrige behandlingen, tenker det blir litt ryddigere API då, og man kan gjenbruke den hvis det skulle være behov. (nesten så det bare er å hente vedtaket til forrige behandling og la frontend ansvare for logikken av å hente ut siste inntekten, eller liste over inntekter)

Lurer også på om det kanskje er litt uklart hvis vi har en endepunkt som gir ett tall, men uten å si hvilken periode det gjelder.
Si eks at man gjør en revurdering bak i tiden, hvilken inntekt skal diffes med hvilken for å gi informasjon om det er endring på 10%?